nerdlandbot/translations/Translations.csv

```diff
@@ -149,6 +149,18 @@
 "privacy_brief","Prints the privacy policy.","Toont de privacy policy die door de bot gehanteerd wordt."
 "privacy_policy_title","Privacy Policy","Privacy Policy"
 "privacy_policy_content","This bot was developed with privacy as one of our core ideals.\nBecause of this we have formulated a few statements about the inner workings.\n\nThe bot will only listen to user commands.\nWe will not parse any user messages, nor track any reactions, unless those made specifically to interact with the bot.\nWe will not track any user data, except for your user ID, as this is required to notify you.\nWe will not store any data about your messages, reactions, or other actions you take on discord.\n\nThe logs we keep are strictly for debugging purposes, and will not contain any personal info.","Bij de ontwikkeling van deze bot hebben we privacy bekeken als een zeer belangrijk punt, om die reden hebben we onze mening hierover uitgedrukt in enkele punten.\n\nDe bot luistert enkel naar commando's die beginnen met de bepaalde prefix.\nDe bot zal nooit berichten die niet met de gegeven prefix beginnen parsen, ook houden we geen reacties bij, behalve de reacties op berichten van de bot.\nDe bot houdt geen gebruikersdata bij, behalve het gebruikers-ID, aangezien dit nodig is om gebruikers te taggen.\nDe bot houdt geen data bij over je berichten, reacties, of andere acties op discord.\n\nDe logs die we bijhouden worden enkel gebruikt om te debuggen. Deze bevatten geen persoonlijke data."
+"reminder_err_too_long","Your timeframe is too long. Please keep the time under 1 day or less than 1440 minutes!","Je tijdsaanduiding is te ver weg, hou de tijd onder 1 dag of minder dan 1440 minuten!"
+"reminder_err_too_short","If you can't remember something for less than 5 minutes, you need help!","Als je iets niet kan herinneren voor minder dan 5 minuten, heb je hulp nodig!"
+"reminder_set_with_message","I will remind you at {0}:{1} of: {2}","Ik zal je herinneren om {0}:{1} aan: {2}"
+"reminder_set_no_message","I will remind you at {0}:{1}.","Ik zal je herinneren om {0}:{1}."
+"reminder_with_message","Hey <@{0}>, you asked me to remind you at {1}:{2} of this: \n {3}","Hey <@{0}>, je hebt me gevraagd je te herinneren om {1}:{2} aan dit: \n {3}"
```

Comment:
Since the user is being tagged just before this message, I think it would be nicer to use the display name here instead of tagging them again (especially since a tag in an embed is not really a tag anyway). Same for the next line.